### PR TITLE
Contact form

### PR DESCRIPTION
**Pull request by massazoum**
Hi team,

I've just submitted a pull request that adds a new field form to the contact section of my portfolio. With this change, users will be able to submit their contact information more easily, making it simpler for potential clients or collaborators to get in touch with me.

I've tested this new feature thoroughly and it seems to be working as expected. Please let me know if you have any issues or concerns about this change.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -239,12 +239,40 @@ <h2 class="CONNECT2"><a href="#">LET'S CONNECT</a> </h2>
...
+        <section class="contact-form">
+
+            <div class="flex-center">
+                <div><h2 class="title-form">Contact me</h2></div>
+            <div><p class="text-contact"> if you have an application you are interested in developing a feature that you need built a project that need coding .I'd love to help with it
+                </p>
+            </div>
+        
+            <div>
+                <form class="form0"  action="https://formspree.io/f/{form_id}" method="post">
```
Comment (bobb-Rob) [inline index.html]: - [ ] I noticed that on click of the `get in touch` button, the form input value does not submit to formspree as expected. it shows _**form not found**_ as seen in the image below 👇🏾 

image

As explained in the project requirement, you need to visit formspree website and create a form there, then copy the `form_id` and use it here 👉🏾 ` <form class="form0"  action="https://formspree.io/f/{form_id}" method="post">`. 
Kindly fix this

File: index.html [hunk 2]
```diff
@@ -239,12 +239,40 @@ <h2 class="CONNECT2"><a href="#">LET'S CONNECT</a> </h2>
...
+
+                    <input type="text" class="inp-name" name="name" id="" maxlength="30" placeholder="Massa zoumanigui"  required>
+
+                    <input type="email" class="inp-email" name="name" id="email" placeholder="zoumaniguimass12@gmail.com"  required>
+
+                    <textarea name="user_message" class="inp-text" id="textarea"  placeholder="Write your message here" maxlength="500" required></textarea>
+
```
Comment (bobb-Rob) [inline index.html:262]: I also notice the following that you can do to improve your project even further.
- Please add `padding-left` to the input elements as indicated by the red arrow in the image below 👇🏾  
- For the textarea element, the padding should be both top and left.
- The `get in touch` button is aligned to the left and not center
- There should be a thick-dark line below the `get in touch` button as observed in the Figma design. Kindly add this to your project.

Your project | Figa Design
---- | ----
image | image

**Review by bobb-Rob — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️
Hi @massazoum, 

Good job so far! 👏👏👏 
There are some issues that you still need to work on to go to the next project but you are almost there!  😃 
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>


## Highlights
✅ No Linter error
✅ Correct use of GitHub flow
✅ Very descriptive Pull Request(PR)
✅ Neat Commit messages

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by osamaashraf6 — APPROVED** [review]
Hi @massazoum , 
# STATUS: APPROVED! 🥇🟢🟢💯 


Your project is complete!  There is nothing else to say other than... it's time to merge it: ship it: 
Congratulations! 🎉 

## Highlights💯🎈🪄✔️
- [x] Good using the formspree✔️
- [x] Good putting the validation for the form✔️
- [x] No linter errors. ✔️
- [x] Good usage in correcting GitHub flow.✔️
- [x] Nice design.✔️


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**